### PR TITLE
fix: swap allowAbsurdFees for maxFeeRate for Dash Core v18

### PR DESCRIPTION
**Pull request by coolaj86**
Feedback wanted: What should the default `maxFeeRate` be?

Re: https://github.com/dashevo/insight-ui/issues/75

### CODE REVIEW

**Review by pshenmic — DISMISSED** [review]
LGTM

File: lib/services/dashd.js [hunk 1]
```diff
@@ -2076,25 +2076,31 @@ Dash.prototype.estimateFee = function(blocks, callback) {
...
  * @param {String|Transaction} transaction - The hex string of the transaction
  * @param {Object=} options
- * @param {Boolean=} options.allowAbsurdFees - Enable large fees
+ * @param {Boolean=} options.maxFeeRate - Cap large fees (0 for unlimited)
  * @param {Function} callback
  */
 Dash.prototype.sendTransaction = function(tx, options, callback) {
   var self = this;
-  var allowAbsurdFees = false;
+  var maxFeeRate = 100000; // duff/kB (typically less than 1000)
```
Comment (thephez) [inline lib/services/dashd.js]: The Dash Core RPC help indicates that it uses 0.1 DASH/kB as the default. Might make sense to stick with that :man_shrugging: 
>maxfeerate      (numeric or string, optional, default=0.10) Reject transactions whose fee rate is higher than the specified value, expressed in DASH/kB

**Review by shumkov — APPROVED** [review]
👍

**Comment by coolaj86** [comment]
This PR is incomplete.

It took a few days for me to set up a full node to test (33gb of blocks and 50gb of indexes). 

Notes for follow up:

`node_modules/@dashevo/dashd-rpc/lib/index.js`:

```diff
- sendRawTransaction: 'str bool bool',
+ sendRawTransaction: 'str float bool',
```

Re: https://github.com/dashevo/dashd-rpc/pull/50

Even after that change I still get:

```txt
Error: "absurdly-high-fee, 383 > 373 (code 256). Code:-26"
```

Can't figure out where it's coming from.

**Comment by coolaj86** [comment]
I updated this from feedback on discord that maxFeeRate was a float, but I'm pretty sure it's supposed to be an int.

When I change from `0.00001000` to `1000` that error goes away.

Also need to investigate `@dashevo/dashcore-node/lib/services/web.js` for socket io stuff...

**Comment by shumkov** [comment]
@coolaj86 so we need one more PR?

**Comment by coolaj86** [comment]
All related PRs have been opened.

I've tested locally going over and under the maximum fee.

**Pull request merged**